{"uid":"d752486a369fd9ac","name":"test_rpc_node_is_synced[Test for Ajuna, url: wss://ajuna.public.curie.radiumblock.co/ws]","fullName":"tests.test_substrate_node_is_synced#test_rpc_node_is_synced","historyId":"7a7f7d089dce426d6651bcba069d9df4","time":{"start":1728400064481,"stop":1728400064481,"duration":0},"status":"broken","statusMessage":"websocket._exceptions.WebSocketBadStatusException: Handshake status 502 Bad Gateway -+-+- {'date': 'Tue, 08 Oct 2024 15:07:45 GMT', 'content-type': 'text/plain; charset=UTF-8', 'content-length': '15', 'connection': 'keep-alive', 'report-to': '{\"endpoints\":[{\"url\":\"https:\\\\/\\\\/a.nel.cloudflare.com\\\\/report\\\\/v4?s=XuVpASlj4gPOXmBZqvMdIsNogv09SUjL2CVHfQfeAB4PLarmaLXXMbpXu9SGP%2FyKa1YmbhCeLljFI7RJH99j1QqJh4CiBDm9UcS00pUBMbeKlyCLInfhzHF2HPqoociLpI66iYIhBMQc3h%2FM1Pszv7vmHg%3D%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}', 'nel': '{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}', 'x-frame-options': 'SAMEORIGIN', 'referrer-policy': 'same-origin', 'cache-control': 'private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0', 'expires': 'Thu, 01 Jan 1970 00:00:01 GMT', 'server': 'cloudflare', 'cf-ray': '8cf70ad52a371316-IAD'} -+-+- b'error code: 502'","statusTrace":"request = <SubRequest 'connection_by_url' for <Function test_rpc_node_is_synced[Test for Ajuna, url: wss://ajuna.public.curie.radiumblock.co/ws]>>\n\n    @pytest.fixture(scope=\"module\", params=collect_nodes_for_chains(get_substrate_chains()), ids=task_ids)\n    def connection_by_url(request):\n        data = request.param\n>       connection = execute_with_timeout(\n            timeout=FIXTURE_TIMEOUT,\n            function=create_connection_by_url,\n            args=(data[\"url\"],),\n            error_message=f'Timeout {FIXTURE_TIMEOUT} when creating connection: {data[\"url\"]}'\n        )\n\ntests/conftest.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/conftest.py:29: in execute_with_timeout\n    return func_timeout(timeout=timeout, func=function, args=args)\n.venv/lib/python3.10/site-packages/func_timeout/dafunc.py:108: in func_timeout\n    raise_exception(exception)\n.venv/lib/python3.10/site-packages/func_timeout/py3_raise.py:7: in raise_exception\n    raise exception[0] from None\nscripts/utils/substrate_interface.py:30: in create_connection_by_url\n    substrate = SubstrateInterface(\n.venv/lib/python3.10/site-packages/substrateinterface/base.py:132: in __init__\n    self.connect_websocket()\n.venv/lib/python3.10/site-packages/substrateinterface/base.py:177: in connect_websocket\n    self.websocket = create_connection(\n.venv/lib/python3.10/site-packages/websocket/_core.py:646: in create_connection\n    websock.connect(url, **options)\n.venv/lib/python3.10/site-packages/websocket/_core.py:261: in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n.venv/lib/python3.10/site-packages/websocket/_handshake.py:65: in handshake\n    status, resp = _get_resp_headers(sock)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nsock = <ssl.SSLSocket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>\nsuccess_statuses = (<HTTPStatus.MOVED_PERMANENTLY: 301>, <HTTPStatus.FOUND: 302>, <HTTPStatus.SEE_OTHER: 303>, <HTTPStatus.TEMPORARY_REDIRECT: 307>, <HTTPStatus.PERMANENT_REDIRECT: 308>, <HTTPStatus.SWITCHING_PROTOCOLS: 101>)\n\n    def _get_resp_headers(sock, success_statuses: tuple = SUCCESS_STATUSES) -> tuple:\n        status, resp_headers, status_message = read_headers(sock)\n        if status not in success_statuses:\n            content_len = resp_headers.get(\"content-length\")\n            if content_len:\n                response_body = sock.recv(\n                    int(content_len)\n                )  # read the body of the HTTP error message response and include it in the exception\n            else:\n                response_body = None\n>           raise WebSocketBadStatusException(\n                f\"Handshake status {status} {status_message} -+-+- {resp_headers} -+-+- {response_body}\",\n                status,\n                status_message,\n                resp_headers,\n                response_body,\n            )\nE           websocket._exceptions.WebSocketBadStatusException: Handshake status 502 Bad Gateway -+-+- {'date': 'Tue, 08 Oct 2024 15:07:45 GMT', 'content-type': 'text/plain; charset=UTF-8', 'content-length': '15', 'connection': 'keep-alive', 'report-to': '{\"endpoints\":[{\"url\":\"https:\\\\/\\\\/a.nel.cloudflare.com\\\\/report\\\\/v4?s=XuVpASlj4gPOXmBZqvMdIsNogv09SUjL2CVHfQfeAB4PLarmaLXXMbpXu9SGP%2FyKa1YmbhCeLljFI7RJH99j1QqJh4CiBDm9UcS00pUBMbeKlyCLInfhzHF2HPqoociLpI66iYIhBMQc3h%2FM1Pszv7vmHg%3D%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}', 'nel': '{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}', 'x-frame-options': 'SAMEORIGIN', 'referrer-policy': 'same-origin', 'cache-control': 'private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0', 'expires': 'Thu, 01 Jan 1970 00:00:01 GMT', 'server': 'cloudflare', 'cf-ray': '8cf70ad52a371316-IAD'} -+-+- b'error code: 502'\n\n.venv/lib/python3.10/site-packages/websocket/_handshake.py:150: WebSocketBadStatusException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":2,"retriesStatusChange":false,"beforeStages":[{"name":"connection_by_url","time":{"start":1728400024228,"stop":1728400039337,"duration":15109},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: wss://ajuna.public.curie.radiumblock.co/ws\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 40, in connection_by_url\n    connection = execute_with_timeout(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 34, in execute_with_timeout\n    raise FixtureTimeoutError(error_message)\n","steps":[{"name":"Function timed out","time":{"start":1728400039337,"stop":1728400039337,"duration":0},"status":"passed","steps":[],"attachments":[{"uid":"2dfb1d7e42569d97","name":"Error","source":"2dfb1d7e42569d97.txt","type":"text/plain","size":79}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},{"name":"connection_by_url","time":{"start":1728400044357,"stop":1728400059464,"duration":15107},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: wss://ajuna.public.curie.radiumblock.co/ws\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 40, in connection_by_url\n    connection = execute_with_timeout(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 34, in execute_with_timeout\n    raise FixtureTimeoutError(error_message)\n","steps":[{"name":"Function timed out","time":{"start":1728400059464,"stop":1728400059464,"duration":0},"status":"passed","steps":[],"attachments":[{"uid":"42abe72b6ddd2bfb","name":"Error","source":"42abe72b6ddd2bfb.txt","type":"text/plain","size":79}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},{"name":"connection_by_url","time":{"start":1728400064481,"stop":1728400065245,"duration":764},"status":"broken","statusMessage":"websocket._exceptions.WebSocketBadStatusException: Handshake status 502 Bad Gateway -+-+- {'date': 'Tue, 08 Oct 2024 15:07:45 GMT', 'content-type': 'text/plain; charset=UTF-8', 'content-length': '15', 'connection': 'keep-alive', 'report-to': '{\"endpoints\":[{\"url\":\"https:\\\\/\\\\/a.nel.cloudflare.com\\\\/report\\\\/v4?s=XuVpASlj4gPOXmBZqvMdIsNogv09SUjL2CVHfQfeAB4PLarmaLXXMbpXu9SGP%2FyKa1YmbhCeLljFI7RJH99j1QqJh4CiBDm9UcS00pUBMbeKlyCLInfhzHF2HPqoociLpI66iYIhBMQc3h%2FM1Pszv7vmHg%3D%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}', 'nel': '{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}', 'x-frame-options': 'SAMEORIGIN', 'referrer-policy': 'same-origin', 'cache-control': 'private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0', 'expires': 'Thu, 01 Jan 1970 00:00:01 GMT', 'server': 'cloudflare', 'cf-ray': '8cf70ad52a371316-IAD'} -+-+- b'error code: 502'\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 40, in connection_by_url\n    connection = execute_with_timeout(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 29, in execute_with_timeout\n    return func_timeout(timeout=timeout, func=function, args=args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/func_timeout/dafunc.py\", line 108, in func_timeout\n    raise_exception(exception)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/func_timeout/py3_raise.py\", line 7, in raise_exception\n    raise exception[0] from None\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/scripts/utils/substrate_interface.py\", line 30, in create_connection_by_url\n    substrate = SubstrateInterface(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/substrateinterface/base.py\", line 132, in __init__\n    self.connect_websocket()\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/substrateinterface/base.py\", line 177, in connect_websocket\n    self.websocket = create_connection(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_core.py\", line 646, in create_connection\n    websock.connect(url, **options)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_core.py\", line 261, in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_handshake.py\", line 65, in handshake\n    status, resp = _get_resp_headers(sock)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_handshake.py\", line 150, in _get_resp_headers\n    raise WebSocketBadStatusException(\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"websocket._exceptions.WebSocketBadStatusException: Handshake status 502 Bad Gateway -+-+- {'date': 'Tue, 08 Oct 2024 15:07:45 GMT', 'content-type': 'text/plain; charset=UTF-8', 'content-length': '15', 'connection': 'keep-alive', 'report-to': '{\"endpoints\":[{\"url\":\"https:\\\\/\\\\/a.nel.cloudflare.com\\\\/report\\\\/v4?s=XuVpASlj4gPOXmBZqvMdIsNogv09SUjL2CVHfQfeAB4PLarmaLXXMbpXu9SGP%2FyKa1YmbhCeLljFI7RJH99j1QqJh4CiBDm9UcS00pUBMbeKlyCLInfhzHF2HPqoociLpI66iYIhBMQc3h%2FM1Pszv7vmHg%3D%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}', 'nel': '{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}', 'x-frame-options': 'SAMEORIGIN', 'referrer-policy': 'same-origin', 'cache-control': 'private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0', 'expires': 'Thu, 01 Jan 1970 00:00:01 GMT', 'server': 'cloudflare', 'cf-ray': '8cf70ad52a371316-IAD'} -+-+- b'error code: 502'","statusTrace":"request = <SubRequest 'connection_by_url' for <Function test_rpc_node_is_synced[Test for Ajuna, url: wss://ajuna.public.curie.radiumblock.co/ws]>>\n\n    @pytest.fixture(scope=\"module\", params=collect_nodes_for_chains(get_substrate_chains()), ids=task_ids)\n    def connection_by_url(request):\n        data = request.param\n>       connection = execute_with_timeout(\n            timeout=FIXTURE_TIMEOUT,\n            function=create_connection_by_url,\n            args=(data[\"url\"],),\n            error_message=f'Timeout {FIXTURE_TIMEOUT} when creating connection: {data[\"url\"]}'\n        )\n\ntests/conftest.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/conftest.py:29: in execute_with_timeout\n    return func_timeout(timeout=timeout, func=function, args=args)\n.venv/lib/python3.10/site-packages/func_timeout/dafunc.py:108: in func_timeout\n    raise_exception(exception)\n.venv/lib/python3.10/site-packages/func_timeout/py3_raise.py:7: in raise_exception\n    raise exception[0] from None\nscripts/utils/substrate_interface.py:30: in create_connection_by_url\n    substrate = SubstrateInterface(\n.venv/lib/python3.10/site-packages/substrateinterface/base.py:132: in __init__\n    self.connect_websocket()\n.venv/lib/python3.10/site-packages/substrateinterface/base.py:177: in connect_websocket\n    self.websocket = create_connection(\n.venv/lib/python3.10/site-packages/websocket/_core.py:646: in create_connection\n    websock.connect(url, **options)\n.venv/lib/python3.10/site-packages/websocket/_core.py:261: in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n.venv/lib/python3.10/site-packages/websocket/_handshake.py:65: in handshake\n    status, resp = _get_resp_headers(sock)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nsock = <ssl.SSLSocket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>\nsuccess_statuses = (<HTTPStatus.MOVED_PERMANENTLY: 301>, <HTTPStatus.FOUND: 302>, <HTTPStatus.SEE_OTHER: 303>, <HTTPStatus.TEMPORARY_REDIRECT: 307>, <HTTPStatus.PERMANENT_REDIRECT: 308>, <HTTPStatus.SWITCHING_PROTOCOLS: 101>)\n\n    def _get_resp_headers(sock, success_statuses: tuple = SUCCESS_STATUSES) -> tuple:\n        status, resp_headers, status_message = read_headers(sock)\n        if status not in success_statuses:\n            content_len = resp_headers.get(\"content-length\")\n            if content_len:\n                response_body = sock.recv(\n                    int(content_len)\n                )  # read the body of the HTTP error message response and include it in the exception\n            else:\n                response_body = None\n>           raise WebSocketBadStatusException(\n                f\"Handshake status {status} {status_message} -+-+- {resp_headers} -+-+- {response_body}\",\n                status,\n                status_message,\n                resp_headers,\n                response_body,\n            )\nE           websocket._exceptions.WebSocketBadStatusException: Handshake status 502 Bad Gateway -+-+- {'date': 'Tue, 08 Oct 2024 15:07:45 GMT', 'content-type': 'text/plain; charset=UTF-8', 'content-length': '15', 'connection': 'keep-alive', 'report-to': '{\"endpoints\":[{\"url\":\"https:\\\\/\\\\/a.nel.cloudflare.com\\\\/report\\\\/v4?s=XuVpASlj4gPOXmBZqvMdIsNogv09SUjL2CVHfQfeAB4PLarmaLXXMbpXu9SGP%2FyKa1YmbhCeLljFI7RJH99j1QqJh4CiBDm9UcS00pUBMbeKlyCLInfhzHF2HPqoociLpI66iYIhBMQc3h%2FM1Pszv7vmHg%3D%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}', 'nel': '{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}', 'x-frame-options': 'SAMEORIGIN', 'referrer-policy': 'same-origin', 'cache-control': 'private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0', 'expires': 'Thu, 01 Jan 1970 00:00:01 GMT', 'server': 'cloudflare', 'cf-ray': '8cf70ad52a371316-IAD'} -+-+- b'error code: 502'\n\n.venv/lib/python3.10/site-packages/websocket/_handshake.py:150: WebSocketBadStatusException","steps":[],"attachments":[{"uid":"3267e745cbd46591","name":"stdout","source":"3267e745cbd46591.txt","type":"text/plain","size":929}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_substrate_node_is_synced"},{"name":"host","value":"fv-az661-64"},{"name":"thread","value":"2143-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_substrate_node_is_synced"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"connection_by_url","value":"{'url': 'wss://ajuna.public.curie.radiumblock.co/ws', 'name': 'Ajuna'}"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"851567725bd12c29","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: wss://ajuna.public.curie.radiumblock.co/ws","time":{"start":1728400044357,"stop":1728400044357,"duration":0}},{"uid":"afeb502a17904f43","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: wss://ajuna.public.curie.radiumblock.co/ws","time":{"start":1728400024227,"stop":1728400024227,"duration":0}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":10,"skipped":0,"passed":58,"unknown":0,"total":68},"items":[{"uid":"aa6739e4cf15a8ee","reportUrl":"https://novasamatech.github.io/test-runner/5755//#testresult/aa6739e4cf15a8ee","status":"passed","time":{"start":1728378476081,"stop":1728378476384,"duration":303}},{"uid":"8cc2ddaffe145379","reportUrl":"https://novasamatech.github.io/test-runner/5754//#testresult/8cc2ddaffe145379","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: wss://ajuna.public.curie.radiumblock.co/ws","time":{"start":1728313704274,"stop":1728313704274,"duration":0}},{"uid":"a0e6ae190313a923","reportUrl":"https://novasamatech.github.io/test-runner/5753//#testresult/a0e6ae190313a923","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: wss://ajuna.public.curie.radiumblock.co/ws","time":{"start":1728292108585,"stop":1728292108585,"duration":0}},{"uid":"885bb7ab38ffe83c","reportUrl":"https://novasamatech.github.io/test-runner/5752//#testresult/885bb7ab38ffe83c","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 502 Bad Gateway -+-+- {'date': 'Sun, 06 Oct 2024 15:08:34 GMT', 'content-type': 'text/plain; charset=UTF-8', 'content-length': '15', 'connection': 'keep-alive', 'report-to': '{\"endpoints\":[{\"url\":\"https:\\\\/\\\\/a.nel.cloudflare.com\\\\/report\\\\/v4?s=0EpCmcGrZQEU4cEWE%2FtRQdBE8roaKyxX0eXZjBNzHQdVKjN7zNAavEd0DNy5n6uYdEqIteV%2FThyOH62JklND7M2i0j4cGnT781d6ZgjFuLiRUhQaZ70LoHKBER5JW6OVkj8v%2BTSdnGez8q0k%2FUFsVSrZVg%3D%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}', 'nel': '{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}', 'x-frame-options': 'SAMEORIGIN', 'referrer-policy': 'same-origin', 'cache-control': 'private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0', 'expires': 'Thu, 01 Jan 1970 00:00:01 GMT', 'server': 'cloudflare', 'cf-ray': '8ce6914b3f0c7be0-LAX'} -+-+- b'error code: 502'","time":{"start":1728227314145,"stop":1728227314145,"duration":0}},{"uid":"26499520f0e7d22d","reportUrl":"https://novasamatech.github.io/test-runner/5751//#testresult/26499520f0e7d22d","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 502 Bad Gateway -+-+- {'date': 'Sun, 06 Oct 2024 09:08:10 GMT', 'content-type': 'text/plain; charset=UTF-8', 'content-length': '15', 'connection': 'keep-alive', 'report-to': '{\"endpoints\":[{\"url\":\"https:\\\\/\\\\/a.nel.cloudflare.com\\\\/report\\\\/v4?s=j0rSSWPgYlx%2FEUx0rLOVsl6Rp0%2FFhiPzUkH2ae4OZE6%2BUIdVIfQkyCVSvT0sCEVQQQq6sybvmUKJq1ZlZ2pu2BqPB6DuWaZiw1vc2jOgqDHzlrvZuxC3J2G54oXqoWs2p%2F0GshEnwpB19bjlgOda5WYXYw%3D%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}', 'nel': '{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}', 'x-frame-options': 'SAMEORIGIN', 'referrer-policy': 'same-origin', 'cache-control': 'private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0', 'expires': 'Thu, 01 Jan 1970 00:00:01 GMT', 'server': 'cloudflare', 'cf-ray': '8ce4815e7e607aaf-SJC'} -+-+- b'error code: 502'","time":{"start":1728205690481,"stop":1728205690481,"duration":0}},{"uid":"8335fccb1350cdde","reportUrl":"https://novasamatech.github.io/test-runner/5750//#testresult/8335fccb1350cdde","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: wss://ajuna.public.curie.radiumblock.co/ws","time":{"start":1728140920071,"stop":1728140920071,"duration":0}},{"uid":"2360db7169309c40","reportUrl":"https://novasamatech.github.io/test-runner/5749//#testresult/2360db7169309c40","status":"passed","time":{"start":1728119318896,"stop":1728119319022,"duration":126}},{"uid":"b2d58a7419e76c0","reportUrl":"https://novasamatech.github.io/test-runner/5748//#testresult/b2d58a7419e76c0","status":"passed","time":{"start":1728054455627,"stop":1728054455880,"duration":253}},{"uid":"b5864854039a6722","reportUrl":"https://novasamatech.github.io/test-runner/5747//#testresult/b5864854039a6722","status":"passed","time":{"start":1728032931392,"stop":1728032931585,"duration":193}},{"uid":"2547dc588237b107","reportUrl":"https://novasamatech.github.io/test-runner/5746//#testresult/2547dc588237b107","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: wss://ajuna.public.curie.radiumblock.co/ws","time":{"start":1727968062297,"stop":1727968062297,"duration":0}},{"uid":"e26c0b93aa5259c1","reportUrl":"https://novasamatech.github.io/test-runner/5745//#testresult/e26c0b93aa5259c1","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when getting properties: wss://ajuna.public.curie.radiumblock.co/ws","time":{"start":1727946457924,"stop":1727946457924,"duration":0}},{"uid":"44d24615e9229f66","reportUrl":"https://novasamatech.github.io/test-runner/5744//#testresult/44d24615e9229f66","status":"passed","time":{"start":1727881629672,"stop":1727881629946,"duration":274}},{"uid":"347fbdd58e32d4cb","reportUrl":"https://novasamatech.github.io/test-runner/5743//#testresult/347fbdd58e32d4cb","status":"passed","time":{"start":1727860042619,"stop":1727860042653,"duration":34}},{"uid":"7f71dc367ec54445","reportUrl":"https://novasamatech.github.io/test-runner/5742//#testresult/7f71dc367ec54445","status":"passed","time":{"start":1727795250063,"stop":1727795250110,"duration":47}},{"uid":"778bfc63e284084d","reportUrl":"https://novasamatech.github.io/test-runner/5741//#testresult/778bfc63e284084d","status":"passed","time":{"start":1727773656840,"stop":1727773656878,"duration":38}},{"uid":"cbd37c2cebb065ea","reportUrl":"https://novasamatech.github.io/test-runner/5740//#testresult/cbd37c2cebb065ea","status":"passed","time":{"start":1727708904317,"stop":1727708904330,"duration":13}},{"uid":"ec0d4079706e3ba5","reportUrl":"https://novasamatech.github.io/test-runner/5739//#testresult/ec0d4079706e3ba5","status":"passed","time":{"start":1727687299191,"stop":1727687299206,"duration":15}},{"uid":"fca682ddf903a58a","reportUrl":"https://novasamatech.github.io/test-runner/5738//#testresult/fca682ddf903a58a","status":"passed","time":{"start":1727622493588,"stop":1727622493633,"duration":45}},{"uid":"48f0eeac09a28ee1","reportUrl":"https://novasamatech.github.io/test-runner/5737//#testresult/48f0eeac09a28ee1","status":"passed","time":{"start":1727600896164,"stop":1727600896210,"duration":46}},{"uid":"17039e64f380e9a8","reportUrl":"https://novasamatech.github.io/test-runner/5736//#testresult/17039e64f380e9a8","status":"passed","time":{"start":1727536030848,"stop":1727536030909,"duration":61}}]},"tags":[]},"source":"d752486a369fd9ac.json","parameterValues":["{'url': 'wss://ajuna.public.curie.radiumblock.co/ws', 'name': 'Ajuna'}"]}